engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/StartPlayingScreen.java

```diff
@@ -85,7 +86,7 @@ public void initialise() {
 
             SimpleUri uri = world.getWorldGeneratorInfo().getUri();
             // This is multiplied by the number of seconds in a day (86400000) to determine the exact  millisecond at which the game will start.
-            float timeOffset = 0.50f;
+            final float timeOffset = 0.50f;
```

Comment:
Unsure if @vampcat meant to make this a *constant* rather than just final. If so it would be all caps and so on as a class level variable :-)